pkg/infra/httpclient/httpclientprovider/datasource_metrics_middleware.go

```diff
@@ -33,10 +37,12 @@ var (
 
 	datasourceResponseHistogram = promauto.NewHistogramVec(
 		prometheus.HistogramOpts{
-			Namespace:                       "grafana",
-			Name:                            "datasource_response_size_bytes",
-			Help:                            "histogram of data source response sizes returned to Grafana",
-			Buckets:                         []float64{128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536, 131072, 262144, 524288, 1048576},
+			Namespace: "grafana",
+			Name:      "datasource_response_size_bytes",
+			Help:      "histogram of data source response sizes returned to Grafana",
+			Buckets: []float64{128, 256, 512, 1 * kb, 2 * kb, 4 * kb, 8 * kb, 16 * kb, 32 * kb, 64 * kb, 128 * kb, 256 * kb, 512 * kb, 1 * mb,
```

Comment:
This have native histograms feature enabled so shouldn't need this update I think, see #87962.

Reply:
Thank you @marefr, closing this!